### ISSUE TITLE
Inappropriately normal TSH

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5866]
**Preferred term label:**
Inappropriately normal TSH

**Synonyms**
see https://omim.org/entry/301035

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
also low free T4

**Comment by pnrobinson** [comment]
Also small thyroid, 9/17  https://jmg.bmj.com/content/55/10/693.long#T1

**Comment by pnrobinson** [comment]
also https://omim.org/entry/301033

**Comment by pnrobinson** [comment]
done
